eZ/Publish/Core/Persistence/Cache/Tests/ContentHandlerTest.php

```diff
@@ -666,6 +783,28 @@ public function testPublish()
             ->expects($this->never())
             ->method('get');
 
+        $cacheItemMock3 = $this->getMock('Stash\Interfaces\ItemInterface');
```

Comment:
Please use `::class` constant.
*(if you feel it's scope-wise inconsistent then you could change other occurrences in this method as well, but I don't mind)* :grimacing: